src/App.js

```diff
@@ -1,24 +1,30 @@
-import logo from './logo.svg';
 import './App.css';
+import { Routes, Route, Link } from 'react-router-dom';
+import Books from './Components/Books';
+import Categories from './Components/Categories';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <nav className="Nav-bar">
+        <h1>
+          Bookstore CMS
+        </h1>
+        <ul className="ul">
+          <li>
+            <Link className="li-app" to="/books"> Books</Link>
+          </li>
```

Comment:
- [ ] Your books page is the homepage or route page of your application means it should not be on a specific URL. Kindly check **to="/books"** to **to="/"** to make it route component :blush: